src/middleware/routers/beneficiaries.js

```diff
@@ -28,7 +27,10 @@ const {
 // GET /v1/beneficiaries
 router.get('/', isLoggedIn, beneficiaries.getAll);
 
+// GET /v1/beneficiaries/:id
+router.get('/:id', beneficiaries.getBeneficiary);
+
 // POST /v1/beneficiaries
-// router.post('/', beneficiaries.create);
+router.post('/', beneficiaries.create);
```

Comment:
Unrelated?

Reply:
except for get ben by id .. will remove create.

Reply:
@herzizza88 can you remove this? There are two identical routes like this ^ .. one with and one without `isLoggedIn`